src/com/facebook/buck/jvm/java/AbstractJavacOptions.java

```diff
@@ -250,6 +256,41 @@ public void appendOptionsTo(
       optionsConsumer.addFlag("proc:none");
     }
 
+    JavacPluginParams standardJavacPluginParams = getStandardJavacPluginParams();
+    if (!standardJavacPluginParams.isEmpty()) {
+      ImmutableList<ResolvedJavacPluginProperties> javacPlugins =
+          standardJavacPluginParams.getPluginProperties();
+
+      // Specify processorpath to search for processors.
+      optionsConsumer.addOptionValue(
+          "classpath",
```

Comment:
None of the other options that are passed to the compiler are constants, I thought of letting the logic remains at is cause it would be strange to add constant for this and not for the others.
And refactoring everything to use constants would only increase the size of this PR with non related to this feature changes

Reply:
`classpath` is only used on the two places where there were changes in it. Also there are tests covering the scenarios

Reply:
But in the end you're right, I'll add it as a constant